src/data/battler-tags.ts

```diff
@@ -2125,6 +2130,34 @@ export class CritBoostTag extends BattlerTag {
   }
 }
 
+/**
+ * A stackable instance of crit boost granted by G-Max Chi Strike
+ */
+export class CritBoostStackableTag extends BattlerTag {
+  public stackCount: number = 0;
+
+  constructor() {
+    super(BattlerTagType.CRIT_BOOST_STACKABLE, BattlerTagLapseType.CUSTOM, 1, Moves.G_MAX_CHI_STRIKE, undefined, true);
```

Comment:
```suggestion
    super(BattlerTagType.CRIT_BOOST_STACKABLE, BattlerTagLapseType.CUSTOM, 1, Moves.G_MAX_CHI_STRIKE, undefined);
```
(To make this tag not passable by Baton Pass)